chainer_chemistry/dataset/preprocessors/megnet_preprocessor.py

```diff
@@ -0,0 +1,422 @@
+import os
```

Comment:
Did you refer some code (original impelmentation)? If you copy some methods from some repository, I think it is better to add reference link like below.
https://github.com/pfnet-research/graph-nvp/blob/master/utils/molecular_metrics.py#L2